protocol/x/clob/keeper/order_state.go

```diff
@@ -221,14 +228,31 @@ func (k Keeper) PruneOrdersForBlockHeight(ctx sdk.Context, blockHeight uint32) (
 				)
 			}
 		}
+		potentiallyPrunableOrdersStore.Delete(it.Key())
```

Comment:
Do we need to prune the whole order `PO/12:` store after we prune all the K:V pairs in it? Or will the KV store automatically get gc'd once there are no keys?

Reply:
No. The prefix store is just a way of conveniently accessing keys with the same prefix. It's the same as querying the underlying store with the prefix prepended.